src/app/home/home.component.spec.ts

```diff
@@ -72,16 +71,6 @@ describe('Component: Home', () => {
 		expect(component).toBeTruthy();
 	}));
 
-	it('should have a title Loklak Search', () => {
```

Comment:
Please add new tests also. You can add it in later patches.

Reply:
After this patch, there will be a patch which is a separate issue to add this created title action in whole project. This will be done later once, its completed.